docs/docs/getting-started/installation/aws.md

```diff
@@ -25,6 +27,30 @@ Apart from the EKS cluster, the installation will create all the necessary AWS r
 - EKS cluster and EKS node groups
 - EC2 instances, as part of the created node group
 
+## Requirements
```

Comment:
I would not do this with a header but rather with a https://docusaurus.io/docs/markdown-features/admonitions